content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -122,29 +122,47 @@ appropriate:
 OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318
 OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
 OTEL_SERVICE_NAME=python-opentelemetry-openai
+OPENAI_API_KEY=<replace_with_your_openai_api_key>
 
 # Set to false or remove to disable log events
 OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT=true
```

Comment:
main thing here is that if you set this to false, it will crash (at least for me)

Reply:
@lzchen @xrmx can one of you double-check the stack trace which happens when this is set to false or unset? I'm not sure if this is already fixed in main or not.

https://github.com/open-telemetry/opentelemetry.io/pull/5575#issuecomment-2475434466

Reply:
https://github.com/open-telemetry/opentelemetry-python/pull/4276 or similar will fix this, though not sure if all backends expect a log with no body or not.